deliver/lib/deliver/commands_generator.rb

```diff
@@ -28,6 +28,13 @@ def deliverfile_options(skip_verification: false)
       return available_options
     end
 
+    def self.force_overwrite_metadata?(options, path)
+      res = options[:force]
+      res ||= ENV["DELIVER_FORCE_OVERWRITE"] # for backward compatibility
+      res ||= UI.confirm("Do you want to overwrite existing metadata on path '#{File.expand_path(path)}'?")
```

Comment:
Fixed. Good catch. I thought `confirm` was doing nothing when in `interactive` but it crashes instead :)